statsmodels/tsa/arima/estimators/hannan_rissanen.py

```diff
@@ -134,7 +134,7 @@ def hannan_rissanen(endog, ar_order=0, ma_order=0, demean=True,
         X_with_fixed_params = lagged_endog[:, params_info.fixed_ar_ix]
         y = endog[max_ar_order:]
         if X_with_fixed_params.shape[1] != 0:
-            y -= X_with_fixed_params.dot(params_info.fixed_ar_params)
+            y = y - X_with_fixed_params.dot(params_info.fixed_ar_params)
```

Comment:
This (along with line 181) is a terrible bug from #7497 (:frowning_face:) where `-=` operator would modify the original endog.